sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -369,7 +369,7 @@ private[sql] class JDBCRDD(
     val sqlText = s"SELECT $columnList FROM $fqTable $myWhereClause"
     val stmt = conn.prepareStatement(sqlText,
         ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY)
-    val fetchSize = properties.getProperty("fetchsize", "0").toInt
+    val fetchSize = properties.getProperty("fetchsize", "10").toInt
```

Comment:
Yes, my mistake. It's due to null properties object.
Is this better to provide a jdbc API like below:
def jdbc(url: String, table: String): DataFrame = {
    jdbc(url, table, new Properties())
  }
I think "properties" meaning is a non essential,so I always use null to test first time.

thanks.